src/linux/helpers/loader.c

```diff
@@ -303,16 +303,17 @@ frida_receive_chunk (int sockfd, void * buffer, size_t length, const FridaLibcAp
       .iov_base = cursor,
       .iov_len = remaining
     };
-    struct msghdr msg = {
-      .msg_name = NULL,
-      .msg_namelen = 0,
-      .msg_iov = &io,
-      .msg_iovlen = 1,
-      .msg_control = NULL,
-      .msg_controllen = 0,
-    };
+    struct msghdr msg;
     ssize_t n;
 
+    /* avoid inline initialization to prevent the compiler attempting to insert a call to  memset */
```

Comment:
Comments should start with the first word capitalized and end in a full stop.

Reply:
Done